src/helpers/searchParam.ts

```diff
@@ -107,7 +111,9 @@ const updateKeyPair = ({
     const updatedValuesString = updatedValuesArray.reduce(
       (result: string[], asset) => [
         ...result,
-        `${asset.assetString}:${getKeyPairString(asset)}`,
+        `${asset.assetString}${URL_ITEM_SEPARATOR_CHAR}${getKeyPairString(
```

Comment:
If `asset.assetString` is `<code>:<issuer pub key>`, what does `getKeyPairString(asset)` return, the home domain key and value?

If `asset.assetString` is just `<code>`, then shouldn't we use `URL_KEY_PAIR_SEPARATOR_CHAR`?

Reply:
This was actually a bug the way we had it before. The item format is `assetString|key>value`. I'll add a comment with the result we're getting for clarity.

We're building query params back, so the format will be `assetSrting|key>value`, `getKeyPairString(asset)` returns `key>value|key1>value1` pairs string.